main/guides/getting-started/syncing-up.md

```diff
@@ -94,6 +94,7 @@ produced a handful of blocks with the `agoric-upgrade-8` upgrade.
 The `agoric-upgrade-9` release requires Go version 1.18 or higher.
 
 1. `git clone https://github.com/agoric/agoric-sdk.git; agoric-sdk && git checkout tags/agoric-upgrade-9`
+1. `cd agoric-sdk`
```

Comment:
```suggestion
1. `git clone https://github.com/agoric/agoric-sdk.git; cd agoric-sdk && git checkout tags/agoric-upgrade-9`
```

Good catch, i am including the `cd` command on the previous line so we still checkout `tags/agoric-upgrade-9`